ares/md/m32x/pwm.cpp

```diff
@@ -11,7 +10,8 @@ auto M32X::PWM::unload(Node::Object parent) -> void {
 }
 
 auto M32X::PWM::main() -> void {
-    n12 clocks = cycle - 1;
+    int clocks = cycle > 0 ? cycle-1 : 4095;
```

Comment:
This change shouldn’t have been necessary, if clocks was 0, being an n5 it should have underflowed and wrapped to 4095 anyway

Reply:
You are correct. I had changed this at some point but I guess it just wound up back where it started.